gcc-vm.yml

```diff
@@ -0,0 +1,147 @@
+---
+- hosts: GCCVM
+  become: true
+  tasks:
+  - name: Install the 'Development tools' package group
+    yum:
+      name: "@Development tools"
+      state: latest
+      update_cache: yes
+
+  - name: Install packages required to create gcc rpm with omnibus
+    yum:
+      name: "{{ item }}"
+      state: latest
+    with_items:
+    - rpm-build
+    - glibc-devel.i686
+    - libgcc.i686
+
+  - name: Remove repository (and clean up left-over metadata)
+    yum_repository:
+      name: wandisco
+      state: absent
+
+  - name: Install wandisco git repository to install git from it
+    yum:
+      name: http://opensource.wandisco.com/centos/6/git/x86_64/wandisco-git-release-6-1.noarch.rpm
+      state: present
+
+- hosts: GCCVM
+  become: true
+  vars:
+          # created with:
+          # python -c 'import crypt; print crypt.crypt("1ns3cur3", "insecure")'
```

Comment:
The formatting is different than in lines 5-6 @ user-creation.yml. Let's stick to one or another.